stdlib/printexc.mli

```diff
@@ -193,7 +193,7 @@ val backtrace_slots : raw_backtrace -> backtrace_slot array option
     recent function call, raise, or primitive [get_backtrace] call in
     the trace.
 
-    Some possible reasons for returning [None] are as follow:
+    Some possible reasons for returning [None] are as follows:
```

Comment:
Um. Guys, I'd like to avoid introducing any controversial stylistic changes as part of this PR, the idea of which is to fix crude mistakes with the least possible effort (and the smallest diff). So, if you can live with `are as follows`, then I suggest that we leave it as it is.